k8s/pkg/apis/longhorn/v1beta2/volume.go

```diff
@@ -99,9 +99,10 @@ type VolumeCloneStatus struct {
 }
 
 const (
-	VolumeConditionTypeScheduled        = "scheduled"
-	VolumeConditionTypeRestore          = "restore"
-	VolumeConditionTypeTooManySnapshots = "toomanysnapshots"
+	VolumeConditionTypeScheduled           = "scheduled"
+	VolumeConditionTypeRestore             = "restore"
+	VolumeConditionTypeTooManySnapshots    = "toomanysnapshots"
+	VolumeConditionTypeWaitForBackingImage = "WaitForBackingImage"
 )
```

Comment:
Do we have any convention of condition value? Just noticed some existing values are just lower case? Should we correct them for consistency?

Reply:
From Kubernetes community, https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties
It should be CamelCase

> Condition type names should make sense for humans; neither positive nor negative polarity can be recommended as a general rule. A negative condition like "MemoryExhausted" may be easier for humans to understand than "SufficientMemory". Conversely, "Ready" or "Succeeded" may be easier to understand than "Failed", because "Failed=Unknown" or "Failed=False" may cause double-negative confusion.

Reply:
Let me fix them in this PR.

Reply:
Fixed.

Reply:
cc @longhorn/qa this change could impact our existing e2e test cases if we use condition for assertion.